Scripts/build-ios-framework.sh

```diff
@@ -21,57 +21,60 @@ echo "Building ${TARGET_NAME}."
 # Install dir will be the final output to the framework.
 # The following line create it in the root folder of the current project.
 PRODUCTS_DIR="${SRCROOT}/../AppCenter-SDK-Apple/iOS"
-
-# The directory to gather all frameworks and build it into xcframework.
-XCFRAMEWORK_DIR="${SRCROOT}/../AppCenter-SDK-Apple/xcframework"
+OUTPUT_DIR="${SRCROOT}/../AppCenter-SDK-Apple/output"
 
 # Working dir will be deleted after the framework creation.
 WORK_DIR=build
-DEVICE_DIR="${WORK_DIR}/Release-iphoneos/"
-SIMULATOR_DIR="${WORK_DIR}/Release-iphonesimulator/"
+OUTPUT_DEVICE_DIR="${OUTPUT_DIR}/${CONFIGURATION}-iphoneos/"
+OUTPUT_SIMULATOR_DIR="${OUTPUT_DIR}/${CONFIGURATION}-iphonesimulator/"
 
 # Make sure we're inside $SRCROOT.
 cd "${SRCROOT}"
 
-# Cleaning previous build.
-xcodebuild -project "${PROJECT_NAME}.xcodeproj" -configuration "Release" -target "${TARGET_NAME}" clean 
-
-# Building both architectures.
-xcodebuild -project "${PROJECT_NAME}.xcodeproj" -configuration "Release" -target "${TARGET_NAME}" -sdk iphoneos
-xcodebuild -project "${PROJECT_NAME}.xcodeproj" -configuration "Release" -target "${TARGET_NAME}" -sdk iphonesimulator
-
-# Cleaning the previous build.
+# Cleaning the previous builds.
 if [ -d "${PRODUCTS_DIR}/${PROJECT_NAME}.framework" ]; then
   rm -rf "${PRODUCTS_DIR}/${PROJECT_NAME}.framework"
 fi
+if [ -d "${OUTPUT_DEVICE_DIR}/${PROJECT_NAME}.framework" ]; then
+  rm -rf "${OUTPUT_DEVICE_DIR}/${PROJECT_NAME}.framework"
+fi
+if [ -d "${OUTPUT_SIMULATOR_DIR}/${PROJECT_NAME}.framework" ]; then
+  rm -rf "${OUTPUT_SIMULATOR_DIR}/${PROJECT_NAME}.framework"
+fi
 
 # Creates and renews the final product folder.
 mkdir -p "${PRODUCTS_DIR}"
 
-# Copy framework.
-cp -R "${DEVICE_DIR}/${PROJECT_NAME}.framework" "${PRODUCTS_DIR}"
+# Create temp directories.
+mkdir -p "${OUTPUT_DEVICE_DIR}"
+mkdir -p "${OUTPUT_SIMULATOR_DIR}"
 
-mkdir -p "${XCFRAMEWORK_DIR}"
+# Clean build.
+xcodebuild -project "${PROJECT_NAME}.xcodeproj" -configuration "${CONFIGURATION}" -target "${TARGET_NAME}" clean
 
-# Copy all framework files to use them for xcframework file creation.
-cp -R "${WORK_DIR}/" "${XCFRAMEWORK_DIR}"
+# Building both architectures.
+xcodebuild -project "${PROJECT_NAME}.xcodeproj" -configuration "${CONFIGURATION}" -target "${TARGET_NAME}" -sdk iphoneos CONFIGURATION_BUILD_DIR="${OUTPUT_DEVICE_DIR}"
+xcodebuild -project "${PROJECT_NAME}.xcodeproj" -configuration "${CONFIGURATION}" -target "${TARGET_NAME}" -sdk iphonesimulator CONFIGURATION_BUILD_DIR="${OUTPUT_SIMULATOR_DIR}"
```

Comment:
Remove `CONFIGURATION_BUILD_DIR="${OUTPUT_DEVICE_DIR}"`, use default setting for it